pages/reference-smart-account/modules/execTransactionFromModuleReturnData.mdx

```diff
@@ -0,0 +1,119 @@
+import { Tabs } from 'nextra/components'
+
+# `execTransactionFromModuleReturnData`
+
+Execute `operation` (0: Call, 1: DelegateCall) to `to` with `value` (Native Token) and return `data`.
```

Comment:
This definition looks a bit weird. Should be something like:

"Executes a transaction from an enabled Safe Module and returns the result data."

References to parameters should be covered in the parameters section.